source/components/patternfly/tests-src/_includes/widgets/communication/alert-danger-dismissable.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger alert-dismissable">
```

Comment:
Nothing under patternfly/tests-src/_includes/widgets should be committed. If you still have a soft link to the widgets directory, please remove it prior to committing.